lib/shared/util.ts

```diff
@@ -11,4 +16,72 @@ export default class Util {
     static indexOf = indexOf;
     static sortBy = sortBy;
     static format = nodeUtil.format;
+    static request = request;
```

Comment:
Template service still uses `require` to get a reference to a request object.

Reply:
Done